README.md

```diff
@@ -942,6 +942,10 @@ Set the following config options to override the help link URLs on the Primary A
 helpLinks: {
   help: 'https://acme.com/help',
   forgotPassword: 'https://acme.com/forgot-password',
+  factorPageCustomLink: {
```

Comment:
all of these, inside `helpLinks` are custom links right? so maybe we don't need to name them "custom*"? Would `factorPage` be enough?

Reply:
Done